docs/docs/backup-and-restore.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Backup and Restore
+---
+
+# Backup and Restore
+
+Cassandra on Mesos supports backup and restore operation. The API for those operation is exposed through REST.
+
+# Configuration
+
+Configuration is provided via environment variable `CASSANDRA_BACKUP_DIRECTORY`. This directory specifies there
+to store named backups. Default values is `backup` directory inside slave's sandbox.
```

Comment:
`Default values is `backup` directory inside slave's sandbox.` -> `The default directory is `backup` inside the task's sandbox.`